### ISSUE TITLE
Screen readers read "Get person credential" button as "Get BCID"

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#749]
Screen readers read "Get person credential" button as "Get BCID" and is missing the "button" label. Should read "Get person credential...button".
